en/arena-tag.json

```
@@ -78,5 +78,7 @@
   "TypeImmuneDamageOverTimeLapseGRASS": "{{pokemonNameWithAffix}} is hurt by G-Max Vine Lash's ferocious beating!",
   "TypeImmuneDamageOverTimeLapseFIRE": "{{pokemonNameWithAffix}} is burning up within G-Max Wildfire's flames!",
   "TypeImmuneDamageOverTimeLapseWATER": "{{pokemonNameWithAffix}} is hurt by G-Max Cannonade's vortex!",
-  "TypeImmuneDamageOverTimeLapseROCK": "{{pokemonNameWithAffix}} is hurt by the rocks thrown out by G-Max Volcalith!"
+  "TypeImmuneDamageOverTimeLapseROCK": "{{pokemonNameWithAffix}} is hurt by the rocks thrown out by G-Max Volcalith!",
+  "sharpSteelOnAdd": "Sharp-pointed pieces of steel\nstarted floating around the opposing Pokemon!",
```

```suggestion
  "sharpSteelOnAdd": "Sharp-pointed pieces of steel\nstarted floating around {{opponentDesc}}!",
```
(Copied from Stealth Rock. This makes the game display "... floating around your team!" if the move is being used against the player.)

---

You have an eagle eye :)